diff --git a/node_modules/@cosmjs/tendermint-rpc/build/tendermint34/tendermint34client.js b/node_modules/@cosmjs/tendermint-rpc/build/tendermint34/tendermint34client.js
index 8f9284a..0f3f345 100644
--- a/node_modules/@cosmjs/tendermint-rpc/build/tendermint34/tendermint34client.js
+++ b/node_modules/@cosmjs/tendermint-rpc/build/tendermint34/tendermint34client.js
@@ -60,7 +60,7 @@ class Tendermint34Client {
         // (our CI) when skipping the status call before doing other queries. Sleeping a little
         // while did not help. Thus we query the version as a way to say "hi" to the backend,
         // even in cases where we don't use the result.
-        const _version = await this.detectVersion(rpcClient);
+        // const _version = await this.detectVersion(rpcClient);
         return new Tendermint34Client(rpcClient);
     }
     static async detectVersion(client) {
diff --git a/node_modules/@cosmjs/tendermint-rpc/build/tendermint35/tendermint35client.js b/node_modules/@cosmjs/tendermint-rpc/build/tendermint35/tendermint35client.js
index d7ea027..e3b36cb 100644
--- a/node_modules/@cosmjs/tendermint-rpc/build/tendermint35/tendermint35client.js
+++ b/node_modules/@cosmjs/tendermint-rpc/build/tendermint35/tendermint35client.js
@@ -65,7 +65,7 @@ class Tendermint35Client {
         // (our CI) when skipping the status call before doing other queries. Sleeping a little
         // while did not help. Thus we query the version as a way to say "hi" to the backend,
         // even in cases where we don't use the result.
-        const _version = await this.detectVersion(rpcClient);
+        // const _version = await this.detectVersion(rpcClient);
         return new Tendermint35Client(rpcClient);
     }
     static async detectVersion(client) {
